mnefun/_mnefun.py

```diff
@@ -1804,7 +1804,7 @@ def _viz_raw_ssp_events(p, subj, ridx):
                      p.n_jobs_resample, projs, None, p.disp_files,
                      method='fft', filter_length=p.filter_length,
                      force_bads=False)
-    raw.plot(events=ev, event_color=colors)
+    raw.plot(events=ev, event_color=colors, block=True)
```

Comment:
So we do want blocking for some functions, and not others, then? Can you add a `block=False` for the one @jyeatman is having trouble with?